openedx/features/calendar_sync/urls.py

```diff
@@ -0,0 +1,16 @@
+"""
+Defines URLs for Calendar Sync.
+"""
+
+
+from django.conf.urls import url
+
+from .views.calendar_sync import CalendarSyncView
+
+urlpatterns = [
+    url(
+        r'^calendar_sync$',
```

Comment:
So this is `<course_url>/calendar_sync`?

Reply:
Yessir